event/event.go

```diff
@@ -0,0 +1,13 @@
+package event
+
+import (
+	"github.com/ipld/go-ipld-prime"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+type Event struct {
+	Version   string
+	Previous  ipld.Link
+	Peer      peer.ID
+	Signature []byte
```

Comment:
Signature is created from the creator DID tp identify the person who is adding this Event?

Reply:
Right. The point of signature is to avoid impersonation attacks. It'll typically be hash of content signed by the peer's public key.